src/coreclr/vm/util.cpp

```diff
@@ -179,8 +182,13 @@ void PrintToStdErrA(const char *pszString) {
 
     HANDLE  Handle = GetStdHandle(STD_ERROR_HANDLE);
 
+#if defined(TARGET_ANDROID)
+    // TODO: make priority configurable?
+    __android_log_write(ANDROID_LOG_INFO, "CoreCLR", pszString);
+#else
     size_t len = strlen(pszString);
     NPrintToHandleA(Handle, pszString, len);
+#endif
 }
```

Comment:
This seems like it should be pushed into the `NPrintToHandleA` function. All calls to `NPrintToHandleA` seem like they should be pushed to `__android_log_write`.

Reply:
Logging changes most probably require a thorough review from all the interested parties, what's in this PR is merely a set of quick hacks to put something in the logcat buffer.